tests/src/pod-smoke.spec.ts

```diff
@@ -0,0 +1,177 @@
+const imageTag = 'latest';
+const containerToRun = 'alpine-container';
+const podToRun = 'alpine-pod';
+
+beforeAll(async () => {
+  pdRunner = new PodmanDesktopRunner();
+  page = await pdRunner.start();
+  pdRunner.setVideoName('pods-e2e');
+  const welcomePage = new WelcomePage(page);
+  await welcomePage.handleWelcomePage(true);
+  // wait giving a time to podman desktop to load up
+  const images = await new NavigationBar(page).openImages();
+  await waitWhile(
+    async () => await images.pageIsEmpty(),
+    5000,
+    1000,
+    false,
+    'Images page is empty, there are no images present',
+  );
+  await deletePod(page, podToRun);
+  await deleteContainer(page, containerToRun);
+});
+
+beforeEach<RunnerTestContext>(async ctx => {
+  ctx.pdRunner = pdRunner;
+});
+
+afterAll(async () => {
+  await deletePod(page, podToRun);
+  await deleteContainer(page, containerToRun);
+  await deleteImage(page, imageToPull);
+  await pdRunner.close();
+}, 90000);
+
+describe.skipIf(process.env.GITHUB_ACTIONS && process.env.RUNNER_OS === 'Linux')(
+  'Verification of pod creation workflow',
+  async () => {
+    test(`Pulling of '${imageToPull}:${imageTag}' image`, async () => {
+      const navigationBar = new NavigationBar(page);
+      let images = await navigationBar.openImages();
+      const pullImagePage = await images.openPullImage();
+      images = await pullImagePage.pullImage(imageToPull, imageTag, 180000);
+
+      const exists = await images.waitForImageExists(imageToPull);
+      expect(exists, `${imageToPull} image is not present in the list of images`).toBeTruthy();
+      await pdRunner.screenshot('pods-pull-image.png');
+    }, 60000);
+
+    test(`Starting a container '${containerToRun}'`, async () => {
+      const navigationBar = new NavigationBar(page);
+      const images = await navigationBar.openImages();
+      const imageDetails = await images.openImageDetails(imageToPull);
+      const runImage = await imageDetails.openRunImage();
+      await pdRunner.screenshot('pods-run-image.png');
+      const containers = await runImage.startContainer(containerToRun);
+      await waitUntil(async () => containers.containerExists(containerToRun), 10000, 1000);
+      await pdRunner.screenshot('pods-container-exists.png');
+      const containerDetails = await containers.openContainersDetails(containerToRun);
+      await waitUntil(async () => {
+        return (await containerDetails.getState()) === ContainerState.Running;
+      }, 5000);
+    });
+
+    test(`Podify container '${containerToRun}'`, async () => {
+      const navigationBar = new NavigationBar(page);
+      const containers = await navigationBar.openContainers();
+      const createPodPage = await containers.openCreatePodPage(containerToRun);
+      await pdRunner.screenshot('pods-creation-page.png');
+      const pods = await createPodPage.createPod(podToRun);
+
+      await pods.heading.waitFor({ state: 'visible', timeout: 6000 });
```

Comment:
you could probably use an assertion, like expect, which eventually does some wait inside. so, `expect(pods.heading).toBeVisible()` should do.

Reply:
This does not work, throws error: `Error: Invalid Chai property: toBeVisible`

Reply:
My bad, use `playExpect`.